src/db.ts

```diff
@@ -1,6 +1,6 @@
-import {createPool, Pool} from 'mysql2';
+import {createPool, Pool, PoolConnection, RowDataPacket} from 'mysql2/promise';
 
-let pool: Pool|null;
+let pool: Pool;
```

Comment:
Or can you be guaranteed that your variable is set? What if the init fails and it is not set? I don't get how TS let you even write this actually but maybe I am missing something.